src/Tizen.Uix.VoiceControlManager/Interop/Interop.VoiceControlCommand.cs

```diff
@@ -0,0 +1,170 @@
+        internal static extern Interop.VoiceControlManager.ErrorCode VcCmdSetCommand(SafeCommandHandle vcCommand, string command);
+
+        [DllImport(Libraries.VoiceControl, EntryPoint = "vc_cmd_get_command")]
+        internal static extern Interop.VoiceControlManager.ErrorCode VcCmdGetCommand(SafeCommandHandle vcCommand, out string command);
+
+        [DllImport(Libraries.VoiceControl, EntryPoint = "vc_cmd_get_unfixed_command")]
+        internal static extern Interop.VoiceControlManager.ErrorCode VcCmdGetUnfixedCommand(SafeCommandHandle vcCommand, out string command);
+
+        [DllImport(Libraries.VoiceControl, EntryPoint = "vc_cmd_set_type")]
+        internal static extern Interop.VoiceControlManager.ErrorCode VcCmdSetType(SafeCommandHandle vcCommand, CommandType type);
+
+        [DllImport(Libraries.VoiceControl, EntryPoint = "vc_cmd_get_type")]
+        internal static extern Interop.VoiceControlManager.ErrorCode VcCmdGetType(SafeCommandHandle vcCommand, out int type);
+
+        [DllImport(Libraries.VoiceControl, EntryPoint = "vc_cmd_set_format")]
+        internal static extern Interop.VoiceControlManager.ErrorCode VcCmdSetFormat(SafeCommandHandle vcCommand, CommandFormat format);
+
+        [DllImport(Libraries.VoiceControl, EntryPoint = "vc_cmd_get_format")]
+        internal static extern Interop.VoiceControlManager.ErrorCode VcCmdGetFormat(SafeCommandHandle vcCommand, out CommandFormat format);
+
+        [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+        internal delegate bool VcCmdListCb(IntPtr vcCommand, IntPtr userData);
+
+        internal sealed class SafeCommandListHandle : SafeHandle
+        {
+            internal bool _ownership;
+
+            public SafeCommandListHandle(IntPtr handle)
+                : base(handle, true)
+            {
+                _ownership = true;
+            }
+
+            public SafeCommandListHandle()
+                : base(IntPtr.Zero, true)
+            {
+                _ownership = true;
+            }
+
+            public override bool IsInvalid
+            {
+                get { return this.handle == IntPtr.Zero; }
+            }
+
+            protected override bool ReleaseHandle()
+            {
+                if (_ownership && !IsInvalid)
+                {
+                    Interop.VoiceControlManager.ErrorCode error = VcCmdListDestroy(this.handle, false);
+                    if (error != Interop.VoiceControlManager.ErrorCode.None)
+                    {
+                        Log.Error(VoiceControlManager.LogTag, "Destroy Failed with error " + error);
+                    }
+                }
+                this.SetHandle(IntPtr.Zero);
+                return true;
+            }
+        }
+
+        internal sealed class SafeCommandHandle : SafeHandle
+        {
+            internal bool _ownership;
+
+            public SafeCommandHandle(IntPtr handle)
+                : base(handle, true)
+            {
+                _ownership = true;
```

Comment:
You do not need to handle ownership
it already impelemented in SafeHandle

https://docs.microsoft.com/ko-kr/dotnet/api/system.runtime.interopservices.safehandle.-ctor?view=netframework-4.7.2